README.md

```diff
@@ -105,50 +107,53 @@ Use [cron-parser](https://github.com/harrisiirak/cron-parser).
 └───────────────────────── second (0 - 59, optional)
 ```
 
-example:
+Example:
+
 ```javascript
-// exce task every 3 hours
+// To execute task every 3 hours
 exports.schedule = {
   type: 'worker',
-  cron: '0 0 */3 * * *',
+  cron: '0 0 */3 * * *'
```

Comment:
@mattma we use eslint `comma-dangle` rule,  so the comma is needed at code. 
anyway, docs is not required